packages/components/label/test/label.spec.ts

```diff
@@ -1,84 +1,81 @@
-import { mount } from '@vue/test-utils'
-import { describe, it, expect } from 'vitest'
-import { faker } from '@faker-js/faker'
-import PuikLabel from '../src/label.vue'
-import type { MountingOptions, VueWrapper } from '@vue/test-utils'
+import { mount, ComponentMountingOptions, VueWrapper } from '@vue/test-utils';
+import { describe, it, expect } from 'vitest';
+import { faker } from '@faker-js/faker';
+import { PuikLabel, LabelProps } from '@prestashopcorp/puik-components';
 
 describe('Label tests', () => {
-  let wrapper: VueWrapper<any>
+  let wrapper: VueWrapper<any>;
 
-  const findLabel = () => wrapper.find('.puik-label')
-  const findLabelOptional = () => wrapper.find('.puik-label--optional')
-  const findLabelRequired = () => wrapper.find('.puik-label--required')
-  const findLabelReadonly = () => wrapper.find('.puik-label--readonly')
+  const findLabel = () => wrapper.find('.puik-label');
+  const findLabelOptional = () => wrapper.find('.puik-label--optional');
+  const findLabelRequired = () => wrapper.find('.puik-label--required');
+  const findLabelReadonly = () => wrapper.find('.puik-label--readonly');
 
   const factory = (
-    propsData: Record<string, any> = {},
-    options: MountingOptions<any> = {}
+    props: LabelProps,
+    options?: ComponentMountingOptions<typeof PuikLabel>
   ) => {
     wrapper = mount(PuikLabel, {
-      props: {
-        ...propsData,
-      },
-      ...options,
-    })
-  }
+      props,
+      ...options
+    } as any);
```

Comment:
``LabelProps`` instead ``any`` ?

Reply:
Nope this also contains the mounting options of the components so it will  crash a TS error, the `any` here was used because of a bug happening with vue-test-utils and typescript but it seems it has been fixed so i removed all the `any` in my last commit

Reply:
Ok, thanks for the explanation.